Formula/mpd.rb

```diff
@@ -39,11 +39,25 @@ class Mpd < Formula
   depends_on "opus"
   depends_on "sqlite"
 
+  uses_from_macos "curl"
+
+  on_linux do
+    depends_on "gcc"
+  end
+
+  fails_with gcc: "5"
+
   def install
     # mpd specifies -std=gnu++0x, but clang appears to try to build
     # that against libstdc++ anyway, which won't work.
     # The build is fine with G++.
-    ENV.libcxx
+    on_macos do
+      ENV.libcxx
+    end
+
+    on_linux do
+      ENV.cxx11
```

Comment:
I wonder if we can just set this unconditionally. Also seems weird to need it on Linux but not on macOS, since the C++ language standard is platform-independent.

Reply:
I think this could be because `gcc-5` doesn't always `std=c++11` by default.  I'll test locally as well to see if it's unneeded.

Reply:
macOS `clang` is supposed to set `-std=gnu++14` by default, except I have reason to think otherwise:
```
❯ cat cplusplus.cpp
#include <iostream>

int main(void)
{
  std::cout << "__cplusplus=" << __cplusplus << std::endl;
  return 0;
}
❯ make cplusplus && ./cplusplus
c++     cplusplus.cpp   -o cplusplus
__cplusplus=199711
```
Makes it seem like they're using `c++98` or something.

Reply:
I was totally off on this - since we are using gcc-10 now, `ENV.cxx11` is not needed because gcc-10 already includes that by default.